malware/get_samples.py

```diff
@@ -71,8 +71,15 @@ def run_query(url):
     try:
         # the response is a 'data' object, which is a list of objects
         resp = json.loads(urlopen(url).read())['data'][0]
-    except Exception as e:
-        print('Failed to load md5: %s' % str(e))
+    except urllib2.HTTPError as e:
+        if e.code==400:
+            error_body = json.loads(e.read())['error']
+            print ('Error message: ',error_body['message'],'for hash: ',error_body['error_data'])
```

Comment:
Can you please use printf formatting like the statement below? (i.e. %s for the strings).